app/src/main/java/com/amrdeveloper/codeviewlibrary/syntax/JavaLanguage.java

```diff
@@ -34,8 +34,8 @@ public class JavaLanguage {
     private static final Pattern PATTERN_ANNOTATION = Pattern.compile("@.[a-zA-Z0-9]+");
     private static final Pattern PATTERN_TODO_COMMENT = Pattern.compile("//TODO[^\n]*");
     private static final Pattern PATTERN_NUMBERS = Pattern.compile("\\b(\\d*[.]?\\d+)\\b");
-    private static final Pattern PATTERN_CHAR = Pattern.compile("'[a-zA-Z]'");
-    private static final Pattern PATTERN_STRING = Pattern.compile("['](.*?)[']|[\"](.*?)[\"]");
+    private static final Pattern PATTERN_CHAR = Pattern.compile("['](.*?)[']|");
```

Comment:
Good, but you should remove the or `|` from the PATTERN_CHAR to be `['](.*?)[']`

Reply:
You can also update the 3 files in one commit no need for 3

Reply:
See now....

Reply:
You need to push new commit to resolve them, they are the same now